client/src/app/core/core-services/openslides.service.ts

```diff
@@ -35,6 +35,10 @@ export class OpenSlidesService {
         return this.booted.value;
     }
 
+    public get isStable(): Promise<void> {
```

Comment:
`isStable` suggest, that it is a boolean. Either:
- make `private stable = new Deferred();` to `public readonly stable = new Deferred();`
- rename it to `waitForStability`

Sidenote: Both suggestions (as well as your) does not prevent a user from doing `this.openSlidesService.isStable.resolve()`, which should be prevented (Like a "read-only" Promise). Since I do not know if this easily doable, it can be skipped ;)

Reply:
Yeah, you're right. I will clean up this, later. :)

Reply:
> Sidenote: Both suggestions (as well as your) does not prevent a user from doing `this.openSlidesService.isStable.resolve()`, which should be prevented (Like a "read-only" Promise). Since I do not know if this easily doable, it can be skipped ;)

This is possible, if a user knows, it is a `Deferred`. But a plain `Promise` can't be resolved out of its scope. :)

Reply:
Ah, right

Reply:
Still a todo